src/platform/qt/ConfigController.cpp

```diff
@@ -365,8 +365,8 @@ void ConfigController::write() {
 	mCoreConfigMap(&m_config, &m_opts);
 }
 
-void ConfigController::makePortable() {
-	mCoreConfigMakePortable(&m_config);
+void ConfigController::makePortable(const char* path) {
+	mCoreConfigMakePortable(&m_config, path);
```

Comment:
For this PR can you hold off on the Qt changes and just pass NULL? We can do a followup PR with Qt changes so the core changes can go in first. It might take a few rounds of bikeshedding the Qt changes for a UI I like.

Reply:
Removed, will submit as a separate PR after.